app/utils/NewPriceUtils.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.products.Price;
+
+import javax.money.MonetaryAmount;
+import java.text.NumberFormat;
+import java.util.Locale;
+import java.util.Optional;
+
+public final class NewPriceUtils {
+
+    private NewPriceUtils() {
+    }
+
+    public static String format(final Price price) {
+        final MonetaryAmount amount = price.getValue();
+        return doFormat(amount);
+    }
+
+    public static String format(final Cart cart) {
+        final MonetaryAmount amount = cart.getTotalPrice();
+        return doFormat(amount);
+    }
+
+    private static String doFormat(MonetaryAmount amount) {
+        final String am = NumberFormat.getInstance(Locale.GERMANY).format(amount.getNumber().doubleValueExact());
```

Comment:
Don't extract the amount this way, you can use this to format money: http://sphereio.github.io/sphere-jvm-sdk/javadoc/v1.0.0-M17/io/sphere/sdk/meta/FormattingDocumentation.html#formatting-money-values